CHANGELOG.md

```diff
@@ -0,0 +1,530 @@
+<a name="2.1.0"></a>
+
+# [2.1.0](https://github.com/vmware/clarity/compare/v2.0.2...2.1.0) (2019-07-25)
+
+### Bug Fixes
+
+* adding labels to datagrid single & expandable column ([dc1dc2b](https://github.com/vmware/clarity/commit/dc1dc2b))
+* change clrSpinner API ([256df22](https://github.com/vmware/clarity/commit/256df22))
+* **datagrid:** input on hideable column ([3b77397](https://github.com/vmware/clarity/commit/3b77397))
+* **forms:** describe by for input validation errors ([730e080](https://github.com/vmware/clarity/commit/730e080)), closes [#3561](https://github.com/vmware/clarity/issues/3561)
+* **login:** add labels for screen readers to form login controls ([6604ba6](https://github.com/vmware/clarity/commit/6604ba6)), closes [#3522](https://github.com/vmware/clarity/issues/3522)
+* **ng:** dropdown item should add disabled attribute if set by input ([bcd032c](https://github.com/vmware/clarity/commit/bcd032c)), closes [#3634](https://github.com/vmware/clarity/issues/3634)
+* **stepper:** remove interstitial content ([c58007e](https://github.com/vmware/clarity/commit/c58007e)), closes [#3560](https://github.com/vmware/clarity/issues/3560)
+* adding the text-edit set to the svg icon build script ([b7d0bef](https://github.com/vmware/clarity/commit/b7d0bef))
+* attempt to fix github PR template ([6bd0fcb](https://github.com/vmware/clarity/commit/6bd0fcb))
+* dropdown item count by screen reader + docs update ([5fd4fd0](https://github.com/vmware/clarity/commit/5fd4fd0))
+* reverting app-level orange alerts to intended color ([#3612](https://github.com/vmware/clarity/issues/3612)) ([a4b63da](https://github.com/vmware/clarity/commit/a4b63da)), closes [#3604](https://github.com/vmware/clarity/issues/3604)
+* Update line-height for clr-control-label ([f8cf498](https://github.com/vmware/clarity/commit/f8cf498))
+
+### Features
+
+* clrSpinner component ([1190d0b](https://github.com/vmware/clarity/commit/1190d0b))
+
+<a name="2.0.2"></a>
+
+## [2.0.2](https://github.com/vmware/clarity/compare/v2.0.1...v2.0.2) (2019-07-05)
```

Comment:
The generator will use the package.json version and commits following the commit guidelines. If it can't find those it will use prior commit tags to generate the version link to github.